diorama.js

```diff
@@ -702,6 +708,38 @@ const createPlayerDiorama = ({
             localVector2.set(0, cameraOffset.y, 0)
               .applyQuaternion(targetQuaternion)
           );
+          // handle player diorama
+          const lerp = (a, b, x) => {
+              return a + (b - a) * x;
+          }
+          if(this.player){

```

Comment:
Is player required to make this look good? If so then it should be required 
 (can be set at construction time) and there is no need for this check?

Also, if there is a `.player` it should be initialized in the constructor regardless, even if it's just to ` null` (for performance reasons).

Reply:
Add `setPlayer`

https://github.com/webaverse/app/blob/32adf742456f9e16d8eab99ce488823362f5c834/diorama.js#L530